lib/Cro/HTTP/ResponseParser.pm6

```diff
@@ -93,6 +105,7 @@ class Cro::HTTP::ResponseParser does Cro::Transform {
                         }
                     }
                     when Body {
+                        $started-body.keep if $started-body.status == Planned;
```

Comment:
I think we'll need a fresh `Promise` in `fresh-message`, otherwise we'll do the wrong thing on persistent connections when the first response is fine, but a later one stalls.